plugins/modules/clustering/znode.py

```diff
@@ -49,6 +49,18 @@
             - Recursively delete node and all its children.
         type: bool
         default: false
+    scheme:
```

Comment:
How about renaming this to `auth_scheme`, and `credential` to `auth_credentials`?